### ISSUE TITLE
notificationCallbackAndroid not defined in PushOptions Interface

### ISSUE THREAD

**Issue by bradmartin** [Kinvey/nativescript-sdk#42]
Kinvey NS SDK: 3.11.6

`notificationCallbackAndroid` doesn't seem to be in the interface `PushOptions` for the android options.

```typescript
// PushOptions interface
interface PushOptions {
  android?: {
    senderID: string
  };
  ios?: {
    alert?: boolean,
    badge?: boolean,
    sound?: boolean
  };
}
```
Referencing the docs here: https://devcenter.kinvey.com/nativescript/guides/push to handle the push notification data.

**Comment by bradmartin** [comment]
Just realized both of the callbacks for `android` and `ios` aren't defined in the interface 👍

**Comment by ramoncardena** [comment]
Any workaround to get the push data on Android?

**Comment by bradmartin** [comment]
The data? I haven't actually had to do any payload stuff. This will work for `register`:

```typescript
Push.register({
  android: {
    senderID: '<Sender ID>',
    notificationCallbackAndroid: (data: string, notification: any) => {
        alert(data);
    }
  },
  ios: {
    alert: true,
    badge: true,
    sound: true,
    notificationCallbackIOS: (message: any) => {
        alert(message);
    }
  }
})
```

If you want the message on android you can get it from the instance of the Firebase notification. I think it was `getBody()` method. 

As for the payload pieces, I think there might be some issues with the current plugin/sdk, @thomasconner is aware of it I believe.

**Comment by bradmartin** [comment]
Ah yes @ramoncardena - here is where I posted about getting the message of the notification right now https://github.com/Kinvey/nativescript-sdk/issues/43#issuecomment-420332971 - but nothing with payload yet

**Comment by ramoncardena** [comment]
@bradmartin yes thanks, I followed your other post and I finally have the data! but I had to modify push.d.ts to include the callback functions in the interface, just wondering if there's any other method not so nasty

**Comment by bradmartin** [comment]
for now you could pack the current published version with your modifications and then use that local version until Kinvey pushes a new release with the updates 👍 - that's better option than modifying node_modules directly, since you'll lose those changes on someone else working on the project or deleting node_modules for builds or anything

**Comment by thomasconner** [comment]
This has been fixed with [3.11.7](https://www.npmjs.com/package/kinvey-nativescript-sdk/v/3.11.7)
